pkg/dynamiccontroller/dynamic_controller.go

```diff
@@ -62,8 +62,10 @@ import (
 	"time"
 
 	"github.com/go-logr/logr"
+	"github.com/kro-run/kro/api/v1alpha1"
```

Comment:
can you please move this import to the L80 import group, checkout this commit message https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e